src/anbox/graphics/multi_window_composer_strategy.cpp

```diff
@@ -47,19 +47,21 @@ std::map<std::shared_ptr<wm::Window>, RenderableList> MultiWindowComposerStrateg
   for (auto &w : win_layers) {
     const auto &renderables = w.second;
     RenderableList final_renderables;
-    auto new_window_frame = Rect::Invalid;
-    auto max_layer_area = -1;
+
+    int32_t new_left   = INT_MAX;
+    int32_t new_top    = INT_MAX;
+    int32_t new_right  = INT_MIN;
+    int32_t new_bottom = INT_MIN;
 
     for (auto &r : renderables) {
-      const auto layer_area = r.screen_position().width() * r.screen_position().height();
       // We always prioritize layers which are lower in the list we got
       // from SurfaceFlinger as they are already ordered.
-      if (layer_area < max_layer_area)
-        continue;
-
-      max_layer_area = layer_area;
-      new_window_frame = r.screen_position();
+      if (r.screen_position().left()   < new_left)   new_left   = r.screen_position().left();
```

Comment:
Can you inherit the code style here the rest of the code uses? 

```
if (r.screen_position().left() < new_left) 
  new_left = r.screen_position().left();
```

Reply:
ok.